src/lib/components/layout/FormLayout/FormLayoutProvider.jsx

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import FormLayoutContext from './FormLayoutContext';
+
+const FormLayoutProvider = ({
+  children,
+  layout,
+}) => (
+  <FormLayoutContext.Provider
+    value={{ layout }}
```

Comment:
Same as above. One liner please.

Reply:
Decided during a phone call with @bedrich-schindler that we do not want to deal with these things on PR. Either there should be an eslint for it or it is at the discretion of the developer.